CIScripts/Tests/CommonTestCode.ps1

```diff
@@ -84,24 +88,26 @@ function Initialize-MPLSoGRE {
            [Parameter(Mandatory = $true)] [System.Management.Automation.Runspaces.PSSession] $Session2,
            [Parameter(Mandatory = $true)] [string] $Container1ID,
            [Parameter(Mandatory = $true)] [string] $Container2ID,
-           [Parameter(Mandatory = $true)] [string] $AdapterName)
+           [Parameter(Mandatory = $true)] [string] $AdapterName,
+           [Parameter(Mandatory = $true)] [string] $VHostName)
```

Comment:
`Test-TCPCommunication` has `$TestConfiguration` param. maybe it should be here as well for the sake of consistency, instead of two params: `$AdapterName` and `$VHostName`?